tests/tacacs/test_ro_user.py

```diff
@@ -20,6 +20,32 @@ def test_ro_user(localhost, duthosts, rand_one_dut_hostname, creds, test_tacacs)
         if fds[0] == "test":
             assert fds[4] == "remote_user"
 
+def test_ro_user_docker_ps(localhost, duthosts, rand_one_dut_hostname, creds, test_tacacs):
```

Comment:
suggest you to combine test_ro_user_docker_ps and test_ro_user_docker_ps_all as a function test_ro_user_sudo_cmds to check all commands that matches the sudoer file.